$NetBSD: patch-ac,v 1.17 2012/09/14 13:20:20 wiz Exp $

Avoid full library name for libGL (including version number) since
it differs between NetBSD's X and pkgsrc.

--- src/video/x11/SDL_x11gl.c.orig	2007-12-31 04:48:13.000000000 +0000
+++ src/video/x11/SDL_x11gl.c
@@ -35,6 +35,9 @@
 #define DEFAULT_OPENGL	"libGL.so.3"
 #elif defined(__OpenBSD__)
 #define DEFAULT_OPENGL	"libGL.so.4.0"
+#elif defined(__NetBSD__)
+/* pkgsrc is .1, base system is .2 */
+#define DEFAULT_OPENGL	"libGL.so"
 #else
 #define DEFAULT_OPENGL	"libGL.so.1"
 #endif
